roles/kubernetes/master/tasks/kubeadm-upgrade.yml

```diff
@@ -37,3 +37,20 @@
     - '"field is immutable" not in kubeadm_upgrade.stderr'
     - kubeadm_upgrade.stdout_lines | length > 1
   notify: Master | restart kubelet
+
+# FIXME: https://github.com/kubernetes/kubeadm/issues/1318
+- name: kubeadm | scale down coredns replicas to 0 if using not using coredns dns_mode
```

Comment:
Same here: `if using not using coredns dns_mode`
Most likely it's "if not using coredns dns_mode"